template.html

```diff
@@ -1,4 +1,2 @@
 <div id="root"></div>
-<script src="//cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react.min.js"></script>
-<script src="//cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react-dom.min.js"></script>
 <script src="{{staticPath}}public/index.js"></script>
```

Comment:
The problem here is that server-side rendering makes this A+B+C scripts to be executed in the correct oder, when instead this html is injected into the page on document.ready via `element.innerHTML` the browser won't guarantee index will be executed after React and React dom will be there. So, render may fail. And actually may succeed (probable when refreshing as all 3 scripts will be in cache already)